doi-transformer/src/test/java/no/unit/nva/doi/transformer/CrossRefConverterTest.java

```diff
@@ -555,7 +553,58 @@ public void toPublicationSetsIsbnWhenCrossrefDocumentIsTypeBook()
                 .map(Isxn::getValue)
                 .map(ISBN_VALIDATOR::validate)
                 .collect(Collectors.toSet());
-        assertEquals(actualValues, expectedValues);
+
+        assertEquals(expectedValues, actualValues);
+    }
+
+    @Test
+    @DisplayName("toPublication filters ISBN error and continues")
```

Comment:
Experience tells me that, in general, a parameterized test, where we throw more real-world data at the ISBN and ISSN validation helps when writing regression tests